deepdrivemd/agents/stream/config.py

```diff
@@ -0,0 +1,109 @@
+from typing import Optional, List, Tuple
+from pydantic import root_validator, validator
+from deepdrivemd.config import AgentTaskConfig
+from pathlib import Path
+
+class OutlierDetectionConfig(AgentTaskConfig):
```

Comment:
This could subclass the sequential config.

Reply:
I am not sure if this is a good idea. The more coupled sequential and streaming components are, the more error prone it would be, the more difficult to work with each component separately without damaging the other version. The config file is trivial, there seems to be no benefit in reusing it. 

I would rather say that the streaming version needs to get rid of a lot of parameters that it does not use, probably 30% of the parameters came from non-streaming version and are not used in streaming version.

Reply:
Thanks for the clarification, in that case I would agree. Can you please remove all the config parameters that are not used in the streaming version?